File: test/cs/case84_comprehension.cs
// このファイルは `test/cs/case84_comprehension.cs` の変換結果（C#サンプル）です。
// Python入力との対応関係を追いやすくするため、日本語コメントを付与しています。
// 仕様変更時は、対応する Python 側ケースと合わせて更新してください。

using System;

public static class Program
{
    public static int comp_like_84(int x)
    {
        List<int> values = /* comprehension */ null;
        return (x + 1);
    }

    public static void Main(string[] args)
    {
        Console.WriteLine(comp_like_84(5));
    }
}

File: test/py/case84_comprehension.py
# このファイルは `test/py/case84_comprehension.py` のテスト/実装コードです。
# 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
# 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。

def comp_like_84(x: int) -> int:
    values: list[int] = [i for i in [1, 2, 3, 4]]
    return x + 1


if __name__ == "__main__":
    print(comp_like_84(5))
